plugins/org.jboss.tools.livereload.core/src/org/jboss/tools/livereload/core/internal/server/jetty/LiveReloadScriptInjectionMiddleManServlet.java

```diff
@@ -83,10 +83,10 @@ public LiveReloadScriptInjectionMiddleManServlet(final String proxyHost, final i
 	@Override
 	protected ContentTransformer newServerResponseContentTransformer(final HttpServletRequest clientRequest,
 			final HttpServletResponse proxyResponse, final Response serverResponse) {
-		final String acceptedContentTypes = clientRequest.getHeader("Accept");
```

Comment:
this also fixes https://issues.jboss.org/browse/JBIDE-15155 - we dont check browser's Accept header - Internet Explorer returns incorrect values (application/x-ms-application, image/jpeg, application/xaml+xml, image/gif, image/pjpeg, application/x-ms-xbap, */*\r\n - notice that there is no html)